ios-base/DroidKaigi 2020/FloorMap/FloorMapViewController.storyboard

```diff
@@ -17,16 +17,18 @@
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
                             <imageView clipsSubviews="YES" userInteractionEnabled="NO" contentMode="scaleAspectFit" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="map" translatesAutoresizingMaskIntoConstraints="NO" id="eba-aI-51s">
-                                <rect key="frame" x="47" y="313" width="320" height="280"/>
+                                <rect key="frame" x="20" y="303.5" width="374" height="299"/>
                                 <constraints>
-                                    <constraint firstAttribute="width" secondItem="eba-aI-51s" secondAttribute="height" multiplier="301:263" id="xzl-2M-4ai"/>
+                                    <constraint firstAttribute="width" secondItem="eba-aI-51s" secondAttribute="height" multiplier="300:240" id="xzl-2M-4ai"/>
```

Comment:
Since the ratio of images that can be obtained from the server side was `1950:1570`,
I changed from `multiplier="301:263"` to `multiplier="300:240"`.